spec/index.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: spec
 title: Specification - Fiscal Data Package
-version: 0.3.0-alpha9
-updated: 28 January 2016
+version: 0.3.0
+updated: 14 March 2016
```

Comment:
Have we really updated the spec? If there was no change we should not change this.